contracts/src/v0.8/ccip/tokenAdminRegistry/RegistryModuleOwnerCustom.sol

```diff
@@ -6,13 +6,16 @@ import {IGetCCIPAdmin} from "../interfaces/IGetCCIPAdmin.sol";
 import {IOwner} from "../interfaces/IOwner.sol";
 import {ITokenAdminRegistry} from "../interfaces/ITokenAdminRegistry.sol";
 
+import {AccessControl} from "../../vendor/openzeppelin-solidity/v5.0.2/contracts/access/AccessControl.sol";
```

Comment:
q: why import the contract instead of the `IAccessControl` interface?

Reply:
It doesn't have DEFAULT_ADMIN_ROLE